components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/http/InboundHttpServerWorker.java

```diff
@@ -106,9 +107,7 @@ public void run() {
                     doPreInjectTasks(axis2MsgContext, (Axis2MessageContext) synCtx, method);
                     boolean executePostDispatchActions =
```

Comment:
executePostDispatchActions need to be renamed to result.

Reply:
renamed in https://github.com/wso2/carbon-mediation/pull/1235/commits/78bc43aee5967b79f73a0479c34bd0fdd6799743